tests/zquantum/qaoa/estimators/cvar_estimator_test.py

```diff
@@ -7,6 +7,14 @@
 
 from zquantum.core.interfaces.estimation import EstimationTask
 
+from zquantum.core.interfaces.estimator_contract import ESTIMATOR_CONTRACT
+
+
+@pytest.mark.parametrize("contract", ESTIMATOR_CONTRACT)
```

Comment:
[not a blocker] The singular `ESTIMATOR_CONTRACT` instead of a plural `ESTIMATOR_CONTRACTS` might be misleading because the reader would have to go to the definition of this constant to see that it's in fact a collection and not a single thing. Do you think we should rename it some time in the future?